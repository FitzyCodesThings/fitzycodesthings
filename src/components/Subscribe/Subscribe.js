// @flow strict
import React from 'react';
import styles from './Subscribe.module.scss';


const Author = () => (
  <div className={styles['subscribe']}>
    <script async data-uid="98430ec254" src="https://skilled-maker-9131.ck.page/98430ec254/index.js"></script>
  </div>
);

export default Author;
