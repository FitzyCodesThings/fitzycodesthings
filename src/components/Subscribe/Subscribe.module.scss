@import '../../assets/scss/variables';
@import '../../assets/scss/mixins';

.subscribe {    
    background: #148f08;
    max-width: $layout-post-width;
    padding: 50px;
    margin: 50px auto;
    text-align: center;
}

.subscribe a {
    color: #fff;
    font-family: Consolas,Monaco,Andale Mono,Ubuntu Mono,monospace;
    font-size: 1.25em;
    font-weight: bold;
}