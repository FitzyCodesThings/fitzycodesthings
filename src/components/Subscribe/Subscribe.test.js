// @flow strict
import React from 'react';
import renderer from 'react-test-renderer';
import { useStaticQuery, StaticQuery } from 'gatsby';
import Subscribe from './Subscribe';
import siteMetadata from '../../../jest/__fixtures__/site-metadata';
import type { RenderCallback } from '../../types';

describe('Subscribe', () => {
  beforeEach(() => {
    StaticQuery.mockImplementationOnce(
      ({ render }: RenderCallback) => (
        render(siteMetadata)
      ),
      useStaticQuery.mockReturnValue(siteMetadata)
    );
  });

  it('renders correctly', () => {
    const tree = renderer.create(<Subscribe />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
