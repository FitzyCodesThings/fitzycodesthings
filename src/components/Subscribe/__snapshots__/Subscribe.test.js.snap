// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscribe renders correctly 1`] = `
<div
  className="subscribe"
>
  <script
    async={true}
    data-uid="98430ec254"
    src="https://skilled-maker-9131.ck.page/98430ec254/index.js"
  />
</div>
`;
